### ISSUE TITLE
Can't upload package to Teams

### ISSUE THREAD

**Issue by kamenstoyanov** [OfficeDev/microsoft-teams-apps-expertfinder#42]
Package never passes the validation tests from https://dev.teams.microsoft.com/appvalidation.html with the following error 
**The file could not be found in the app package. Learn to fix it**. 
![image](https://user-images.githubusercontent.com/90390061/132675307-1a364bc3-978d-4759-9877-20b0cb3b11b3.png)
Tried to change the manifest version also. Even included the our Sharepoint site domain to the validDomains section.

**Comment by kamenstoyanov** [comment]
Please update us on the issue. Still when I try to update in Teams admin center. The error is
![image](https://user-images.githubusercontent.com/90390061/139268522-88ea45af-a092-4ab9-8ecd-a42da36fc62b.png)

**Comment by Hodsha** [comment]
Hay, did you solve the problem?

**Comment by Rutger-Knijnenburg** [comment]
Hi! I had the same issue and was able to fix it.
I have opened a PR with the required change in order to upload the manifest.

tl;dr: Rename the file 'zn-TW.json' to 'zh-TW.json'. :)
